src/OmniSharp.Roslyn.CSharp/Services/SemanticHighlight/SemanticHighlightService.cs

```diff
@@ -107,6 +107,7 @@ class ClassifiedResult
                 [ClassificationTypeNames.VerbatimStringLiteral] = SemanticHighlightClassification.VerbatimStringLiteral,
                 [ClassificationTypeNames.StringEscapeCharacter] = SemanticHighlightClassification.StringEscapeCharacter,
                 [ClassificationTypeNames.ClassName] = SemanticHighlightClassification.ClassName,
+                [ClassificationTypeNames.RecordName] = SemanticHighlightClassification.ClassName,
```

Comment:
The other option would be to map a separate classification for record names and have the clients deal with it. Which is fine, just need to know what to do.

Reply:
no, this is fine IMHO

Reply:
Well, the only concern is that we'll need to do something different when struct records come out, but that's a concern for next year :).